### DISCUSSION TITLE
Change Text Orientation or rotate test 90 Degrees using FaBGL

### DISCUSSION THREAD

**Discussion by 22kashaf-khan** [fdivitto/FabGL#253] [Q&A] [answered]
I want to display some text in Portrait mode. I'm using a VGA Display. Since we can't rotate with a function. I have decided to place each component like in any portrait mode.
What should I change in the code to rotate it 90 degrees?
void init()
{
  canvas.setBrushColor(Color::Black);
    canvas.clear();
    canvas.setGlyphOptions(GlyphOptions().FillBackground(true));
    canvas.selectFont(&fabgl::FONT_8x8);
    canvas.setPenColor(Color::BrightWhite);
    canvas.setGlyphOptions(GlyphOptions().DoubleWidth(1));
    canvas.drawText(50, 15, "BonuS");
    canvas.setGlyphOptions(GlyphOptions().DoubleWidth(0));
}

**Comment by fdivitto** [comment, marked as answer]
I'm sorry, there isn't a way to rotate text or any other canvas object. You should create a new font with glyphs already rotated, then dispose (draw) glyphs vertically, one at the time.
